Other/MergeShops/Adam1a1Merge.cs

```diff
@@ -0,0 +1,239 @@
+                        Core.HuntMonster("djinn", "Lamia", "Tasty Poison", 10);
+                        Core.HuntMonster("charredpath", "Toxic Wisteria", "Necessary Antidote");
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "IOU Slip":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    Core.RegisterQuests(8009);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("vendorbooths", "Caffeine Imp", "Coffee Beans", 10);
+                        Core.HuntMonster("djinn", "Lamia", "Tasty Poison", 10);
+                        Core.HuntMonster("charredpath", "Toxic Wisteria", "Necessary Antidote");
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "EctoBlade":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("shadowfallwar", "Skeletal Fire Mage", "EctoBlade", isTemp: false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Ectoplasmic Chains":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    Core.RegisterQuests(8010);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("temple", "Doomwood Ectomancer", "Refined Ectoplasm", 10);
+                        Core.HuntMonster("ectocave", "Ektorax", "Ektorax's Ectoplasm");
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Bongo Cart Pet":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    if (!Bot.Quests.IsUnlocked(8013)) {
+                        Adam1a1Quest.doall();
+                    }
+                    Core.AddDrop("Love Token");
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.EnsureAccept(8013);
+                        // Red Rose
+                        if (!Core.CheckInventory("Red Rose")){
+                            while(!Bot.ShouldExit && !Core.CheckInventory(21567, 10)){
```

Comment:
missing a { above this line to complete the if.. which idt is needed here? im confused what your trying todo... if the buy requires an item and the item is just a hunt do a 
while(should edit && !Core.check(req.name, quant)
{
hunt line here
buy line here
}

Reply:
ill prolly have to look at the item ignore this for now looks like alota stuff in ifs for an item ._.